lib/util.rb

```diff
@@ -99,4 +99,14 @@ def self.aws_tag_specifications(resource_type, name, additional_tags = {})
   def self.monitor_process?
     ENV["MONITOR_PROCESS"] == "1"
   end
+
+  def self.calculate_ips_v4
+    Address
+      .select { set_masklen(:cidr, 16).as(:masked) }
+      .where { (family(:cidr) =~ 4) & (masklen(:cidr) !~ 32) }
+      .distinct
+      .order(:masked)
```

Comment:
I'm not sure how it sorts but jfyi

https://github.com/ubicloud/ubicloud/pull/3475#discussion_r2157332248